### PR TITLE
Various edits, technical corrections & additions

### PR DESCRIPTION
**Pull request by hansieodendaal**

### CODE REVIEW

File: proposals/181029-overview.md [hunk 1]
```diff
@@ -143,17 +142,18 @@ tolerance and thus promising a truly permissionless second layer.
 
 _Other tools in our belt / possible features_:
 * 2nd layer nodes use public/private keys to identify themselves.
-* 2nd layer nodes post a bonded contract that they stand to lose if they get caught cheating
+* 2nd layer nodes post a bonded contract that they stand to lose if they get caught cheating or not performing
```
Comment (CjS77) [inline proposals/181029-overview.md]: Maybe we should just say "proven to be bad actors", since we haven't defined what cheating or performing would mean yet.

File: proposals/181105-terms.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ this glossary to disambiguate ideas, and work towards a
 [ubiquitous language](https://blog.carbonfive.com/2016/10/04/ubiquitous-language-the-joy-of-naming/) for this project.  
 
 ## Base layer
-[Base Layer]: #Base-layer 'The Tari layer handling payments and secured by proof of work'
+[Base Layer]: #Base-layer "The Tari layer handling payments and secured by proof of work"
```
Comment (CjS77) [inline proposals/181105-terms.md]: Why the formatting wars already? Leave the single quotes _si vous plait_

File: proposals/181107-base-layer-architecture.md [hunk 3]
```diff
@@ -155,11 +155,25 @@ This entails that any and all terms defined in the [Glossary] will have a softwa
...
   * verifying all the range proofs in the `Transaction`
 * If a new `Transaction` is valid,
-  * Mark it as verified (is this how we want to handle this???)
-  * Push the Verified Transaction onto the internal `MessageBus`
+
+  Mark it as verified (is this how we want to handle this???)
+
+  Push the Verified Transaction onto the internal `MessageBus`
+
+#### Compile new [Block] by subscribing to the `MessageBroker` as a `VerifiedTransactionSubscriber`
```
Comment (CjS77) [inline proposals/181107-base-layer-architecture.md]: This is substantive new content, so should be a different PR

File: proposals/181029-overview.md (hunk 1 above)
Comment (hansieodendaal) [reply proposals/181029-overview.md]: The concept I am thinking about is that the asset issuer's investment must be protected, So a node can still pose harm to the investment even when not bad acting, like being offline unintentional for a long time. So if a "bad actor" is anything that harm the investment then that should cover it.

File: proposals/181105-terms.md (hunk 2 above)
Comment (hansieodendaal) [reply proposals/181105-terms.md]: I think my editor did that automatically, can't think why I would change it if the popups were working.

File: proposals/181107-base-layer-architecture.md (hunk 3 above)
Comment (hansieodendaal) [reply proposals/181107-base-layer-architecture.md]: Will do

File: proposals/181105-terms.md (hunk 2 above)
Comment (CjS77) [reply proposals/181105-terms.md]: Can you see where your editor is changing this and change that setting or disable auto-formatting?

File: proposals/181107-base-layer-architecture.md (hunk 3 above)
Comment (CjS77) [reply proposals/181107-base-layer-architecture.md]: 👍

File: proposals/181029-overview.md (hunk 1 above)
Comment (CjS77) [reply proposals/181029-overview.md]: +1

**Comment by CjS77** [comment]
Closing this PR, since all the typo edits have been folded in already, and the new content can be included as a separate PR.
